packages/components/src/box-control/README.md

```diff
@@ -105,6 +105,14 @@ If this property is true, a button to reset the box control is rendered.
 - Required: No
 - Default: `true`
 
+### isGroupedDirections
```

Comment:
Nit: For this to be more semantic english, should we use `has` instead of `is` here given `directions` is plural?

An even more descriptive prop name might be like `splitOnAxis` or something like that. Grouped directions could mean corners are grouped together for example.

```suggestion
### hasGroupedDirections
```

Reply:
Thanks, I love some good naming tips, the `is` never sit right with the plural for me. I actually quite like `splitOnAxis`, so I've updated this PR to use that instead. Happy to rename again if you think there's a better option.